user/protontricks/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "protontricks"
+pkgver = "1.11.1"
+pkgrel = 0
+build_style = "python_pep517"
+make_build_env = {"SETUPTOOLS_SCM_PRETEND_VERSION": pkgver}
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-setuptools_scm",
+]
+_pydeps = [
+    "python-pillow",
+    "python-setuptools",
+    "python-vdf",
+]
+depends = [
+    "virtual:cmd:winetricks!winetricks",
+    "virtual:cmd:zenity!zenity",
+    *_pydeps,
+]
+checkdepends = [
+    "bash",
+    "python-pytest",
+    *_pydeps,
+]
```

Comment:
```suggestion
depends = [
    "winetricks",
    "zenity",
    <the pydeps i am too lazy to type>
]
checkdepends = [
    "bash",
    "python-pytest",
    *depends,
]
```

a little extra hostdep install via check is ok, saves on needing the whole variable

for the virtual, imo it doesn't make sense here because it's very clearly 'winetricks and zenity' as the deps wholesale